frontend/src/components/PrevNext.js

```diff
@@ -1,16 +1,52 @@
 import React from 'react';
 import Data from './data.json'
-
+import './styles/BodyStyle.css'
+
+function openCloseAnswer(e)
+{
+  let element = e.target;
+  console.log(element)
+  if(element.tagName.toLowerCase() !== "div")
+    element = element.parentElement
+  element = element.nextElementSibling
```

Comment:
am seeing what you try to do here it will be nice with a small explicit comment of what you are trying to do !